Maskbook/Scene/App/FIleService/View/FileServiceView.swift

```diff
@@ -19,20 +21,35 @@ struct FileServiceView: View {
                 }
             } else {
                 // file list view
-                UploadFileList(viewModel: viewModel)
+                fileList(with: proxy)
                     .ignoresSafeArea(.container, edges: .bottom)
+                    .onAppear {
+                        viewModel.isVisible = true
+                    }
             }
         }
-        .overlay(
-            Group {
-                if viewModel.showOnboard || viewModel.isUploading {
-                    Color.clear
-                } else {
-                    uploadButton
-                }
-            },
-            alignment: .bottomTrailing
+        // keep it here to avoid container size change
+        // and keep uploadingItemList's position
+        .ignoresSafeArea(.keyboard)
+    }
+
+    // MARK: Private
+
+    @StateObject
+    private var viewModel: FileServiceViewModel
+
+    private let buttonSize: CGFloat = 56
+
+    private var cardGradient: some View {
+        LinearGradient(
+            colors: [
+                Asset.Colors.Gradient.f2F8Ff.asColor(),
```

Comment:
Please give the color a normal name

Reply:
This is good for Gradient color name space. As gradient always come with a random hex name.

Reply:
A normal name helps us  easyly recognise the usage of color.
Almost every color has a normal folder and name.
If more and more color become this random hex color, It will be very hard for us to reuse them.